src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,59 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
 
+public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        writeFile(toFileName, readFile(fromFileName));
+        System.out.println(readFile(fromFileName));
+    }
+
+    private String readFile(String fromFileNameData) {
+        StringBuilder stringBuilder;
+        String[] oneLine;
+        final String supply = "supply";
+        final String buy = "buy";
+        int supplySum = 0;
+        int buySum = 0;
+        int result = 0;
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileNameData));
+            String value = bufferedReader.readLine();
+
+            while (value != null) {
+                oneLine = value.split(",");
+                if (oneLine[0].equals(supply)) {
+                    supplySum += Integer.parseInt(oneLine[1]);
+                } else if (oneLine[0].equals(buy)) {
+                    buySum += Integer.parseInt(oneLine[1]);
+                } else {
+                    throw new RuntimeException("There is nothing in file");
```

Comment:
```suggestion
                    throw new RuntimeException("Unknown operation");
```
якщо ми заййшли в цикл while - це означає, що файл не пустий, тому тут краще підкорегувати меседж